ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/ConsistentPackageContentView.kt

```diff
@@ -8,8 +8,21 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
 import com.revenuecat.purchases.ui.revenuecatui.UIConstant
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallViewState
```

Comment:
@NachoSoto I think this file is missing in the branch

Reply:
Same with PaywallViewOptions and InternalPaywallView. I suspect they are just renames?

Reply:
Oh yes, we renamed a bunch of files in https://github.com/RevenueCat/purchases-android/pull/1351. This probably needs fixing all those references. It's just renames